src.ts/contract/contract.ts

```diff
@@ -304,14 +304,19 @@ function buildWrappedMethod<A extends Array<any> = Array<any>, R = any, D extend
         try {
             result = await runner.call(tx);
         } catch (error: any) {
-            if (isCallException(error) && error.data) {
-                throw contract.interface.makeError(error.data, tx);
+            if (error.data) {
```

Comment:
When is this true, but the error is not a CallException? My concern with this is that random weird things thrown by a node might accidentally get trapped by this and bad data bubble up.